.github/workflows/build.yml

```diff
@@ -164,10 +164,14 @@ jobs:
         working-directory: python
         run: python -m cibuildwheel --output-dir dist
         env:
-          # uncomment to stop building separate wheels for pypy
-          # CIBW_SKIP: "pp*"
+          # don't build separate wheels for pypy
+          CIBW_SKIP: "pp*"
           CIBW_ENVIRONMENT: 'PATH="$HOME/.cargo/bin:$PATH"'
           CIBW_ENVIRONMENT_WINDOWS: 'PATH="$UserProfile\.cargo\bin;$PATH"'
+          # first find the xcode version: https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md
+          #   (currently 14.3)
+          # then find the deployment target: https://developer.apple.com/support/xcode/
+          CIBW_ENVIRONMENT_MACOS: 'MACOSX_DEPLOYMENT_TARGET=10.13'
```

Comment:
> first find the xcode version

This is one release announcement. Is there a changelog we should look at for the latest?

> then find the deployment target: https://developer.apple.com/support/xcode/

Assuming this means the "Xcode 14.3* macOS Ventura 13" line? May mention prepending the version with "10.".